apps/storybook/.storybook/global.css

```
@@ -1,3 +1,11 @@
+@font-face {
+    font-family: 'Arimo';
```

Stylelint (17.12.0)

[error] Expected no quotes around "Arimo" (font-family-name-quotes)

(font-family-name-quotes)